### ISSUE TITLE
Add python3 support

### ISSUE THREAD

**Issue by elnappo** [toolswatch/vFeed#62]
from 2to3:
```
RefactoringTool: Files that need to be modified:
RefactoringTool: ./api_calls.py
RefactoringTool: ./vfeedcli.py
RefactoringTool: ./config/constants.py
RefactoringTool: ./config/stats.py
RefactoringTool: ./lib/common/banner.py
RefactoringTool: ./lib/common/database.py
RefactoringTool: ./lib/common/utils.py
RefactoringTool: ./lib/core/search.py
RefactoringTool: ./lib/core/update.py
RefactoringTool: ./lib/core/methods/exploit.py
RefactoringTool: ./lib/core/methods/info.py
RefactoringTool: ./lib/core/methods/json_dump.py
RefactoringTool: ./lib/core/methods/patches.py
RefactoringTool: ./lib/core/methods/ref.py
RefactoringTool: ./lib/core/methods/risk.py
RefactoringTool: ./lib/core/methods/rules.py
RefactoringTool: ./lib/core/methods/scanners.py
```

these are mostly replaced print statements

**Comment by toolswatch** [comment]
Let me check this issue with the next API release.  Sounds nice :)

Thanks .

**Comment by elnappo** [comment]
I think I've already done with this, see https://github.com/toolswatch/vFeed/compare/master...elnappo:python3?expand=1

**Comment by toolswatch** [comment]
Awesome bro !!
I was deep into >> http://python-future.org/compatible_idioms.html
It will be merged with the next release :)

Thanks a million ;)

**Comment by elnappo** [comment]
I've fixed #61 with https://github.com/toolswatch/vFeed/commit/e8abfb763ca7bba3ebecccd27d7be0cae4b00b5e in my python3 branch. Furthermore I also added a .gitignore file. Should I open a pull request for these changes (including python3 support)?

**Comment by toolswatch** [comment]
Added in 0.7
